components/create_comment/create_comment.jsx

```diff
@@ -953,7 +960,7 @@ export default class CreateComment extends React.PureComponent {
                                 ref='textbox'
                                 disabled={readOnlyChannel}
                                 characterLimit={this.props.maxPostSize}
-                                preview={this.state.preview}
+                                preview={this.state.showPreview}
```

Comment:
Textbox link (Preview | Edit message) is not updating.
```suggestion
                                showPreview={this.state.showPreview}
```

Same for CreatePost and EditPostModal.